__tests__/app.test.js

```diff
@@ -0,0 +1,31 @@
+const request = require("supertest");
+const app = require("../app");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const testData = require("../db/data/test-data");
+
+beforeEach(() => {
+    return seed(testData);
+});
+
+afterAll(() => {
+    db.end();
+});
+
+describe("GET /api/topics", () => {
+    test("200: should respond with an array of all topic objects", () => {
+        return request(app)
+            .get("/api/topics")
+            .expect(200)
+            .then(({ body }) => {
+                const { topics } = body;
+                expect(topics).toBeInstanceOf(Array);
```

Comment:
Would be ideal to see a length check here, if the response came back as an empty array this expect statement would pass and the forEach would never run giving you an auto pass. So we need to guard against this with a length check to be certain there is content in there and the forEach will run.